core/src/main/java/software/amazon/awssdk/http/ExecutionContext.java

```diff
@@ -87,6 +87,7 @@ public ExecutionAttributes executionAttributes() {
      *
      * @return true if retry capacity was consumed
      */
+    @ReviewBeforeRelease("Not used. Should it be?")
```

Comment:
IIRC that's stored in the RetryableStage now so this can be removed

https://github.com/aws/aws-sdk-java-v2/blob/master/core/src/main/java/software/amazon/awssdk/http/pipeline/stages/RetryableStage.java#L123